service/src/main/java/com/codeforcommunity/ServiceMain.java

```diff
@@ -33,13 +33,20 @@
 import com.codeforcommunity.propertiesLoader.PropertiesLoader;
 import com.codeforcommunity.requester.Emailer;
 import com.codeforcommunity.rest.ApiRouter;
+import com.codeforcommunity.rest.IpThrottlingFilter;
+import io.github.bucket4j.Bandwidth;
+import io.github.bucket4j.BucketConfiguration;
+import io.github.bucket4j.Refill;
 import io.vertx.core.Vertx;
+import java.time.Duration;
 import java.util.Properties;
 import org.jooq.DSLContext;
 import org.jooq.impl.DSL;
 
 public class ServiceMain {
   private DSLContext db;
+  public static final long CAPACITY = 20;
+  public static final Refill REFILL = Refill.greedy(10, Duration.ofMinutes(1));
```

Comment:
❓ How'd you determine the initial capacity and refill rate - are these some common numbers or just gut feelings? 

IMO we might want to bump up either the capacity or the refill rate, since there are some actions that might occur in quick succession such as approving/rejecting/uploading images which are one request per image (tbh it might be a good idea to switch to bulk routes here, even though it was my idea to do it one at a time)

Reply:
These are arbitrary gut feeling values. wdym by bulk routes? do you mean router specific instead of global?